packages/core/src/localization/en/en.json

```diff
@@ -267,7 +267,7 @@
     "BiometricsUnlock": "Unlock with biometrics",
     "IncorrectPIN": "Incorrect PIN",
     "RepeatPIN": "Please try your PIN again.",
-    "IncorrectPINTries": "Incorrect PIN: {{tries}} tries before timeout",
+    "IncorrectPINTries": "Incorrect PIN: {{tries}} tries before your app is temporarily locked",
```

Comment:
I think because this line changed in English we should reset it in French and Brazilian Portuguese until someone can translate it otherwise the text will drift over time.